MPSGameLift/Code/Source/Utils/MPSMatchmakingComponent.cpp

```diff
@@ -163,10 +163,11 @@ namespace MPSGameLift
         AZStd::string actualRegion;
         AWSCore::AWSResourceMappingRequestBus::BroadcastResult(actualRegion, &AWSCore::AWSResourceMappingRequests::GetDefaultRegion);
 
+        AZStd::string restApi;
+        AWSCore::AWSResourceMappingRequestBus::BroadcastResult(restApi, &AWSCore::AWSResourceMappingRequests::GetResourceNameId, "MPSMatchamking");
         config->region = actualRegion.c_str();
-        // TODO: Set this up correctly;
         config->endpointOverride =  AZStd::string::format("https://%s.execute-api.%s.amazonaws.com/%s",
-            "", actualRegion.c_str(), "requestmatch").c_str();
+            restApi.c_str(), actualRegion.c_str(), "Prod/requestmatchmaking").c_str();
```

Comment:
do we need to be calling `/requestmatchstatus` somewhere too, when checking on the status of the submitted request for a mtach? (maybe that's in this branch already and I haven't found it)

Reply:
Its not in this branch ;)